src/main/java/org/casbin/jcasbin/main/CoreEnforcer.java

```diff
@@ -709,4 +709,8 @@ public boolean isAutoNotifyDispatcher() {
     public void setAutoNotifyDispatcher(boolean autoNotifyDispatcher) {
         this.autoNotifyDispatcher = autoNotifyDispatcher;
     }
+
+    public boolean mustUseDispatcher() {
```

Comment:
```suggestion
    protected boolean mustUseDispatcher() {
```

Reply:
Hi @nodece, great suggestion, Thank you.

Fixed.